app/cdap/components/WranglerV2/SectionWrapper/index.tsx

```diff
@@ -0,0 +1,146 @@
+
+export interface ISectionWrapperComponentProps {
+  columnsSelected: IHeaderNamesList[];
+  transformationName: string;
+  transformationInfoLink: string;
+}
+
+const SubTitle = styled(Typography)`
+  &&& {
+    color: ${grey[700]};
+    font-weight: 400;
+    font-size: 14px;
+    letter-spacing: 0.1px;
+    line-height: 22px;
+  }
+`;
+
+const StyledButton = styled(Button)`
+  &&& {
+    color: #3367d6;
+    float: left;
+    font-size: 15px;
+    font-weight: 400;
+    letter-spacing: 0.46px;
+    line-height: 26px;
+    margin-top: 20px;
+    margin-bottom: 30px;
+    text-transform: none;
+    width: 162px;
+  }
+`;
+
+const InfoIconComponent = styled(InfoOutlinedIcon)`
+  color: #3367d6;
+  cursor: pointer;
+  font-size: 20px;
+  height: 20px;
+  margin-left: 5px;
+  margin-top: 13px;
+  width: 20px;
+`;
+
+const TextInfoIconWrapper = styled.section`
+  align-items: center;
+  display: flex;
+  padding: 0px;
+  margin-bottom: 20px;
+`;
+
+const SectionBodyText = styled(Typography)`
+  &&& {
+    color: ${grey[700]};
+    font-size: 16px;
+    font-weight: 400;
+    letter-spacing: 0.5px;
+    line-height: 24px;
+    margin-top: 10px;
+  }
+`;
+
+const StyledDivider = styled(Divider)`
+  &&& {
+    margin-top: 16px;
+  }
+`;
+
+const ColumnName = styled(Typography)`
+  &&& {
+    color: ${grey[700]};
+    font-size: 16px;
+    font-weight: 400;
+    letter-spacing: 0.5px;
+    line-height: 24px;
+    margin-top: 10px;
+    margin-bottom: 20px;
+  }
+`;
+
+export const PREFIX = 'features.WranglerNewUI.AddTransformationStepPanel';
+
+export default function SectionWrapper({
+  columnsSelected,
+  transformationName,
+  transformationInfoLink,
+}: ISectionWrapperComponentProps) {
+  return (
+    <>
+      <SubTitle>
+        {!Boolean(columnsSelected.length) && T.translate(`${PREFIX}.noColumnsSelected`)}
+        {Boolean(columnsSelected.length) &&
+          `${columnsSelected.length}  ${T.translate(`${PREFIX}.columnsSelected`)}`}
+      </SubTitle>
+      <StyledDivider />
+      <SectionComponent title="Function" showDivider={true} showTickIcon={true}>
+        <TextInfoIconWrapper>
+          <SectionBodyText component="span">{transformationName}</SectionBodyText>
+          <a href={transformationInfoLink}>
+            <InfoIconComponent />
+          </a>
+        </TextInfoIconWrapper>
+      </SectionComponent>
+      <SectionComponent
+        title={T.translate(`${PREFIX}.selectColumnTitle`).toString()}
+        showDivider={true}
+        showTickIcon={Boolean(columnsSelected.length)}
+      >
+        <SubTitle>{T.translate(`${PREFIX}.selectColumnSubTitle`)}</SubTitle>
+        {Boolean(columnsSelected.length) &&
+          columnsSelected.map((selectedColumn, index) => {
+            return (
+              <ColumnName component="span">{`${index + 1}. ${selectedColumn.label}`}</ColumnName>
+            );
+          })}
+        {!Boolean(columnsSelected.length) && (
```

Comment:
Boolean is not required here as length starts from 1
and 0 is considered false anyway

Reply:
removed Boolean method here as it not necessary